editor/src/clj/editor/dialogs.clj

```diff
@@ -787,21 +787,37 @@
 
 (defn- sanitize-common [name]
   (-> name
-      string/trim
       (string/replace #"[/\\]" "") ; strip path separators
       (string/replace #"[\"']" "") ; strip quotes
-      (string/replace #"[<>:|?*]" ""))) ; Additional Windows forbidden characters
+      (string/replace #"[<>:|?*]" "") ; Additional Windows forbidden characters
+      string/trim))
```

Comment:
We actually want to trim last, otherwise we still might end up with white space on the edges of a name after we remove the forbidden characters, e.g. for cases like `aaa  '`.